app/models/users_social_profile.rb

```diff
@@ -0,0 +1,28 @@
+class UsersSocialProfile < ApplicationRecord
+  belongs_to :user
+
+  delegate :uid, to: :user, prefix: false
```

Comment:
Eventually we'll allow multiple profiles per user and will need a `uid` per profile, so can we move `uid` to this new table now?  uids are more related to a given identity provider than with the user, so this feels more logical, and I see you already included `uid` in the migration creating this table so I'm guessing you agree ;-)